docs/rfcs/024-ingress-egress-metrics.md

```diff
@@ -0,0 +1,64 @@
+# Ingress-Egress Metrics
+
+# Goals
+* Track ingress/egress traffic directly from a compute node, which may be sending stuff due to 
+  a custom extension.
+* Lay the groundwork for further traffic management on the compute node. 
+
+# Background
+As we move towards allowing custom extensions on the compute node, we need to cope with the fact
+that these extensions are untrusted. They could send or receive things from the open internet or
+even try to send bogus requests around on the internal network. Although we currently use the proxy
+to measure incoming/outgoing traffic, custom extensions will potentially be able to interact with
```

Comment:
The initial idea was to limit compute node networking to pageservers/safekeepers (and proxy) only. Any changes there ?

Reply:
Yes, we want to allow them access to/from the internet for such features in addition to what you've mentioned

1. extensions (pg_fdw, dblink, http)
2. logical replication from neon 
3. compute as a replica for external DB (under discussion yet)

Reply:
why not leverage the cloud's provider virtual networking for such purpose?
for example, use security groups to allow only specific ports and protocols and if needed enforce the upper layer with firewall ?